msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,322 @@
 
-   * **Make your message concise and informative**. The user experience and the knowledge of your app will vary. A user may have used your app on another platform or know nothing about your app. You want to tailor your message to all audiences and in a couple sentences explain what your bot does and the ways to interact with it. You must also explain the value of the app and how the users will benefit from using it.
-![Cafe and Dinning bot](../../../../assets/images/faq/cafe-bot.png)
+### 9.2 In-meeting experience
 
-* **Make your message actionable**. Think about the first thing you want users to do after installing your app. Is there a cool command they must try? Is there another onboarding experience they must know about? Do they need to sign-in? You can add actions on an adaptive card or provide specific examples such as *“Try asking….”*, *“This is what I can do…”*.
+* Apps must [only use a dark theme](~/apps-in-teams-meetings/design/designing-apps-in-meetings.md#theming) during meetings.
+* A tooltip should display the app name when hovering over the app icon during meetings.
+* Messaging extensions must function the same during meetings as they do outside meetings.
 
-#### Welcome messages in the team or channel  scope
+### 9.3 In-meeting tabs
 
-Things are a little bit different when the bot is first added to a channel. Normally, you shouldn't send a 1:1 message to everyone on the team, but the bot can send a welcome message in the channel.
+* Must be responsive and adapt within the provided space. Make sure to maintain padding and component sizes.
+* Must have a back button if there is more than one layer of navigation.
+* Must not include more than one dismiss or close button. This may confuse users since there's already a built-in header button in the header to dismiss the tab.
+* Must not have horizontal scrolling.
 
-### &#9989; Mobile responsiveness, no direct upsell or payment
+### 9.4 In-meeting dialogs
 
-* Your tabs, adaptive cards, bot messages and content in task modules must be responsive for a variety of mobile device screens sizes.
-* Apps that support iOS must be fully functional on the latest iPad device using the latest version of iOS.
-* Must not include any direct references to in-app purchases, trial offers, offers for paid versions, or links to any online stores where users can purchase or acquire other content, apps, or add-ins from your Teams app on mobile OS (Android, iOS).
-* The iOS or Android version of the add-in must not show any UI or language or link to any other apps, add-ins, or website that ask the user to pay.
-* The associated Privacy Policy and Terms of Use pages must also be free of any commerce UI or Store links.
+* Should be used sparingly and for scenarios that are light and task-oriented.
+* Must display content in a single column and not have multiple navigation levels.
+* Must not use task modules.
+* Should align with the center of the meeting stage.
```

Comment:
Just checking: Is this “should” and not “must”?

Reply:
Must

Reply:
```suggestion
* Must align with the center of the meeting stage.
```